web/templates/precall.ejs

```diff
@@ -68,8 +68,8 @@
     </form>
     <div class="info-links">
       <div class="faq-link">
-        <span>Need help? Read our</span>
-        <a href="../Vonage-Free-Conferencing-help.pdf" target="_blank">FAQs</a>
+        <span>Need help? </span>
```

Comment:
The span tag no longer does anything. We should remove them in both pages. Also, should we make all of "Need help? Read our FAQs" a link (not just "Read or FAQs")?